thirdeye-server/src/main/java/ai/startree/thirdeye/core/AppAnalyticsService.java

```diff
@@ -124,4 +127,19 @@ public ConfusionMatrix computeConfusionMatrixForAnomalies() {
     }
     return matrix;
   }
+
+  private List<AnomalyFeedback> getAnomalyFeedbacks() {
+    return anomalyManager.findParentAnomaliesWithFeedback().stream()
+        .map(MergedAnomalyResultDTO::getFeedback)
+        .collect(Collectors.toList());
+  }
+
+  public AnomalyStatsApi computeAnomalyStats() {
+    List<AnomalyFeedback> feedbacks = getAnomalyFeedbacks();
+    AnomalyStatsApi stat = new AnomalyStatsApi()
+        .setTotalCount(anomalyManager.countParentAnomalies())
+        .setCountWithFeedback((long) feedbacks.size());
+    feedbacks.forEach(feedback -> stat.incFeedbackStatCount(feedback.getFeedbackType()));
```

Comment:
I would just build the map here and set it on the POJO. so we don't have to even initialize it.

Motivation here is to keep business logic classes and data classes separate. so data classes like API classes are completely dumb! only job is to move around the resource classes and get serialized to json.

Reply:
Makes sense. Done.